jmh-benchmarks/src/main/java/org/apache/kafka/jmh/metadata/MetadataRequestBenchmark.java

```diff
@@ -192,7 +193,8 @@ private KafkaApis createKafkaApis() {
             null,
             brokerFeatures,
             new FinalizedFeatureCache(brokerFeatures),
-            null);
+            null,
+            new QuotaCache());
```

Comment:
nit: maybe `null` to be consistent with `brokerMetadataListener` being passed as `null`?